spec/schemas/_common.mapping.yaml

```
@@ -929,7 +933,8 @@ components:
       enum:
         - recursive
         - term
-    PointProperty:
+    XyPointProperty:
+      x-version-added: 2.4
```

```suggestion
      # x-version-added: 2.4
```

This causes #558, I'll fix it but for now it will work around the test issue.